club/theme.json

```diff
@@ -53,6 +53,10 @@
             "gap": {
                 "horizontal": "min(30px, 5vw)",
                 "vertical": "min(30px, 5vw)"
+            },
+            "fontSize": {
+                "default": "clamp(0.875rem, calc(0.875rem + ((1vw - 0.48rem) * 0.7212)), 1.25rem)",
+                "x-large": "clamp(1.25rem, calc(1.25rem + ((1vw - 0.48rem) * 2.4038)), 2.5rem)"
```

Comment:
What is the difference between the fontSize preset versus this custom one? Why are there two `x-large` sizes, and this one is responsive? 🤔